members/schema.py

```diff
@@ -101,9 +106,12 @@ class Meta:
 class Query(object):
     profiles = graphene.List(ProfileObj, token=graphene.String(required=True))
     profile = graphene.Field(ProfileObj, username=graphene.String(required=True), token=graphene.String(required=True))
-
+    GetTodaysLeaves = graphene.List(LeaveRecordObj, token=graphene.String(required=True))
```

Comment:
Create a more generic getLeaveRecords API and pass in the date, instead of making a api just for today.